src/CONST.ts

```diff
@@ -2557,6 +2557,12 @@ const CONST = {
     TRANSLATION_KEYS: {
         ATTACHMENT: 'common.attachment',
     },
+    TEACHERS_UNITE: {
+        PUBLIC_ROOM_ID: '795574440662771',
+        POLICY_ID: 'B795B6319125BDF2',
+        POLICY_NAME: 'Teachers Unite!',
```

Comment:
```suggestion
        POLICY_NAME: 'Expensify.org / Teachers Unite!',
```

Policy has been renamed